src/main/resources/google_checks.xml

```diff
@@ -288,11 +288,13 @@
                     METHOD_DEF, QUESTION, RESOURCE_SPECIFICATION, SUPER_CTOR_CALL, LAMBDA,
                     RECORD_DEF"/>
     </module>
+    <!-- Absence of COLON token for OperatorWrap explained in https://github.com/checkstyle/checkstyle/issues/4122 -->
```

Comment:
After thinking on this a bit, it is better to place this comment in `AllChecksTest.java` :
https://github.com/checkstyle/checkstyle/blob/8bacab1adcae21ac4e2e7c131eace8995fad782f/src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java#L248-L251   We should replace the current comment (L249-L250) with yours, then remove `TYPE_EXTENSION_AND` token from the set to show that CI will pass since token is now in the config.

Reply:
@strkkk did travis not catch this `until..` comment because it is mutli-line?

Reply:
@nmancus1 no, it will not catch it, since job is pretty stupid and does not detect multiline comments

Reply:
@nmancus1, thanks for pointing this out, I didn't know about this file. I have slightly altered and moved the comment to this one.

Reply:
Resolved.

Reply:
@bossever FWIW, I think it would be a good practice to `grep -R "<issue-number>"` from source root as part of our workflow, I plan to do this going forward.